app/assets/javascripts/live_search.js

```diff
@@ -150,6 +150,16 @@
         });
       });
     }
+
+    var $results = $('.finder-results');
+    if ($results.length > 0) {
+      if ($results.find('.document--top .document__link').length == 1) {
```

Comment:
if you refactor the `.document__link` in the scss file then you could refactor this to `.find('.document__link--top')`

Reply:
`$results.find('.document--top .document__link')` seems to be used quite a few times. Could I suggest you cache the elements  eg `var $topResultsLink = $results.find('.document__link--top')`